### DISCUSSION TITLE
How do you become qualified for the leaderboard?

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1460] [Questions] [answered]
I know is this an exact duplicate of #1203 but I didn't get an exact statistic. @Miodec

**Comment by ShizukoV** [comment, marked as answer]
I think you're asking **Miodec** but I'll answer anyways lol. **_For me_**, to get on the leaderboard all I have to do is **_increase my best wpm for either my 15s wpm or my 60s wpm._** For example, in the 15s mode in the global leaderboard, if 999th place is 170 wpm, to get placed on the leaderboard, all you must do is get 170.01 wpm or more. If it's the daily leaderboard it's the same, except it resets everyday.

This answer is pretty much the same as the selected answer on #1203 lol...

You asked "I was wondering how you become qualified for the leaderboard; is there a average wpm or accuracy you need?" on #1203...

There is no average wpm or accuracy that is required, (at least that I know of), all you need is a wpm score which is greater than the lowest score on the leaderboard.

That means if you're fast enough you **_should/must_** be on the leaderboard...
And if you aren't fast enough, then you **_won't_** be on the leaderboard.

I kinda hoped that answered your question, because there isn't any sort of "exact statistic" you need to be on the leaderboard...

**Reply by Miodec** [reply]
Simply getting a new pb will not get you on the leaderboard. You have to beat the lowest score currently on the leaderboard, and there are limited spots.

**Reply by ShizukoV** [reply]
I know lol, that's why I bolded "For me" at the start, since it may not imply to him/her.
As for the limited spots I wrote "999th" since for the global leaderboards, that's the lowest "position" you can obtain (I think).
